application/dash_application/views/single_lane_ts.py

```diff
@@ -88,16 +90,18 @@
 
 def get_bamqc_data():
     bamqc_df = util.get_bamqc3_and_4()
+    bamqc_df = util.df_with_fastqc_data(bamqc_df, [BAMQC_COL.Run, BAMQC_COL.Lane, BAMQC_COL.Barcodes])
+
     bamqc_df[special_cols["Total Reads (Passed Filter)"]] = round(
         bamqc_df[BAMQC_COL.TotalReads] / 1e6, 3)
```

Comment:
Stray `BAMQC_COL`

Reply:
Unsure what you mean by this. Those are the columns to merge on.

Reply:
Fixed